app/src/ui/add-repository/write-default-readme.ts

```diff
@@ -3,7 +3,10 @@ import * as Path from 'path'
 
 const DefaultReadmeName = 'README.md'
 
-function defaultReadmeContents(name: string): string {
+function defaultReadmeContents(name: string, description?: string): string {
+  if (description) {
```

Comment:
This could be cleaned-up by using a ternary expression:

```ts
return description !== undefined ? `# ${name}\n ${description}` : `# ${name}\n`
```

_Note, I added the explicit check to `undefined` because we generally try to not to rely on falsy values._

Reply:
ah good point